scripts/mongo/createTables.php

```diff
@@ -38,7 +38,7 @@ function generateTables($id, $tableId,$dbName)
 
 if ($argc!=3 && $argc!=4 && $argc!=5)
 {
-    echo "usage: ./createTables.php tripodConfig.json dbName [tableId] [_id]\n";
+    echo "usage: ./createTables.php tripodConfig.json storeName [tableId] [_id]\n";
```

Comment:
How do you feel about dragging this (and createViews.php) script into the modern world with getopts?  I understand if you don't want to complicated this PR though.